src/main/java/org/jenkinsci/plugins/proccleaner/PsCleaner.java

```diff
@@ -64,7 +69,34 @@ public void clean(CleanRequest request) throws IOException, InterruptedException
         }
 
         try {
-            killer.kill(System.getProperty("user.name", ""), request.getListener().getLogger());
+            String osName = System.getProperty( "os.name" ).toLowerCase();
+            String className = null;
+            String methodName = "getUsername";
+            String user = null;
+
+            // https://issues.jenkins-ci.org/browse/JENKINS-53739
```

Comment:
all the stuff bellow won't work on e.g. IBM java, but I guess good for now and we can fix it when some issue for IBM java is looged :-)

Reply:
Sure, but in that case the old way is used or I've overlooked sth.?

Reply:
no, you are right, I just pointed out that it doesn't fix the issue for all JDKs.

Reply:
JNA (Core uses it too) provides a way how to do similar thing - probably more portable... WDYT?

Reply:
depends on you if you want to spend time with it. IMHO IBM JDK is not used very often and I consider it as a very minor bug and low priority